tests/admin_changelist/tests.py

```diff
@@ -672,9 +672,16 @@ def test_get_admin_log_templatetag_custom_user(self):
 class SeleniumFirefoxTests(AdminSeleniumWebDriverTestCase):
 
     available_apps = ['admin_changelist'] + AdminSeleniumWebDriverTestCase.available_apps
-    fixtures = ['users.json']
     webdriver_class = 'selenium.webdriver.firefox.webdriver.WebDriver'
 
+    @classmethod
+    def setUpTestData(cls):
+        User.objects.create(
+            pk=100, username='super', first_name='Super', last_name='User', email='super@example.com',
+            password='sha1$995a3$6011485ea3834267d719b4c801409b8b1ddd0158', is_active=True, is_superuser=True,
+            is_staff=True, last_login=datetime.datetime(2007, 5, 30, 13, 20, 10),
+            date_joined=datetime.datetime(2007, 5, 30, 13, 20, 10))
```

Comment:
I looked at the log and realized those tests are run with `USE_TZ=False` so everything should be good here.